### PR TITLE
fix command load fixtures

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

**Comment by veolys-nanne** [comment]
:+1:

File: en/developer_guide/install.rst [hunk 1]
```diff
@@ -110,11 +110,12 @@ Launch the grunt command to generate all assets
 Load the fixtures
 -----------------
 
-Open Orchestra comes with a set of fixtures for a quick demo tour
+Open Orchestra needs some fixtures to work.
```
Comment (ngilain) [inline en/developer_guide/install.rst]: I think you should talk of a production environment here. And can you give some examples to give the reader an idea of what is discussed here ?

**Comment by alavieille** [comment]
@itkg-ngilain pr updated

**Comment by ngilain** [comment]
:+1:

**Pull request merged**